index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      defer
+      src="https://kit.fontawesome.com/75e02ba484.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Twitter UI Clone</title>
+  </head>
+  <body>
+    <div class="wrapper tablet desktop">
+      <!-- rh-sb section -->
+      <div class="rh-sb">
+        <div class="rh-sb__search-b">
+          <div class="search">
+            <i class="fas fa-search"></i>
+            <input
+              type="text"
+              class="searchTwitter"
+              placeholder="Search Twitter"
+            />
+          </div>
+        </div>
+        <div class="rh-sb__news">
+          <div class="news__heading">
+            <h5>What's Happening</h5>
+          </div>
+          <div class="news-item news-item-1__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="news-item news-item-2__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="news-item news-item-3__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="news-item news-item-4__text">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti
+              eius doloremque totam nihil commodi accusamus.
+            </p>
+          </div>
+          <div class="image-container__news-1">
+            <img
+              class="boolean-logo"
+              src="../boolean-uk-html-boolean-image-grid/assets/images/boolean.jpeg"
```

Comment:
There is something a little bit wrong with how you're stating the `src` for the images here. It'll work on your local machine, however, when it comes to viewing this on a different platform (codesandbox for instance) you rimages won't load. to fix this, you would have to change your image `src` to something like `./assets/images/boolean.jpeg`.